### PR TITLE
Redemptions fixes

### PR DESCRIPTION
**Pull request by thomas-waite**
# Redemptions fixes

### CODE REVIEW

File: proposals/dao/tip_121c.ts [hunk 1]
```diff
@@ -67,6 +66,23 @@ const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: Named
...
+  await contracts.daiHoldingPCVDeposit.connect(guardianSigner).pause();
+
+  // 3. Can NOT revoke PCV_CONTROLLER_ROLE from DAI PCV drip controller - role admin is GOVERNOR
+
+  // 4. Slay fuseWithdrawalGuard on PCV Sentinel
+  await contracts.pcvSentinel.slay(addresses.fuseWithdrawalGuard);
+
+  //////// Verify daiHoldingPCVDeposit paused  ///////
+  // - needs to be paused up until DAO vote execution time
+  expect(await contracts.daiHoldingPCVDeposit.paused()).to.be.true;
```
Comment (thomas-waite) [inline proposals/dao/tip_121c.ts:84]: Check it is paused here as we will remove steps 1 - 4 once guardian has done actions. 

Need to ensure it is paused for DAO vote to execute and the DAO vote will unpause() it to withdraw funds - so can't just check in validate()

File: proposals/description/tip_121c.ts [hunk 2]
```diff
@@ -15,6 +15,16 @@ const DAO_TIMELOCK_LQTY_BALANCE = '1101298805118942906652299';
 const tip_121c: TemplatedProposalDescription = {
   title: 'TIP-121c: Final redemptions',
   commands: [
+    // 0. Unpause the DAI Holding PCV deposit
+    {
```
Comment (thomas-waite) [inline proposals/description/tip_121c.ts:19]: Guardian will pause it before execution so oldPsm.allocateSurplus() can't be called. Needs to be unpaused in this DAO vote so we can transfer funds from it to the psm

File: proposals/dao/tip_121c.ts [hunk 3]
```diff
@@ -23,7 +24,6 @@ const AMOUNT_FEI_MINTED_BY_E2E = toBN('10000000000000000000000000'); // 10M
 
 /////////////  Tribe Redeemer config
 // Circulating amount of TRIBE, which is redeemable for underlying PCV assets
-// TODO: Update with final circulating TRIBE figure
 const REDEEM_BASE = ethers.constants.WeiPerEther.mul(458_964_340);
```
Comment (xklob) [inline proposals/dao/tip_121c.ts:26]: This number hasn't changed; was it supposed to?

Comment (thomas-waite) [reply proposals/dao/tip_121c.ts:26]: No, it's fixed and is an invariant

Comment (thomas-waite) [reply proposals/dao/tip_121c.ts:26]: It's calculated by tribe.totalSupply() - a bunch of tribe locked in various places like Core

File: proposals/dao/tip_121c.ts [hunk 4]
```diff
@@ -23,19 +23,17 @@ const AMOUNT_FEI_MINTED_BY_E2E = toBN('10000000000000000000000000'); // 10M
...
 
 // Lido deposit balance, being withdrawn and sent to Tribe Redeemer
 const STETH_DEPOSIT_BALANCE = toBN('50296523674661485703301');
 const DAO_TIMELOCK_FOX_BALANCE = toBN('15316691965631380244403204');
 const DAO_TIMELOCK_LQTY_BALANCE = toBN('1101298805118942906652299');
 
-// User circulating Fei as determined by fei-tools.com for block 15589242
-const USER_CIRCULATING_FEI_AT_FIXED_BLOCK = ethers.constants.WeiPerEther.mul(58_708_166);
+// User circulating Fei as determined by fei-tools.com for block 15591696
+const USER_CIRCULATING_FEI_AT_FIXED_BLOCK = ethers.constants.WeiPerEther.mul(58_641_830);
```
Comment (thomas-waite) [inline proposals/dao/tip_121c.ts:34]: Used for testing/verification purposes only

**Review by xklob — APPROVED** [review]

**Pull request merged**